src/lib/sql/sql_pipeline_statement.cpp

```diff
@@ -116,8 +119,9 @@ const std::shared_ptr<SQLQueryPlan>& SQLPipelineStatement::get_query_plan() {
     throw std::runtime_error("Error while translating query plan:\n  " + std::string(exception.what()));
   }
 
-  if (_use_mvcc) {
-    // If we need a transaction context but haven't passed one in, this is the latest point where we can create it
+  if (_control_block->use_mvcc == ChunkUseMvcc::Yes) {
```

Comment:
I haven't yet decided if I like this explicit bool thing or not. This is a good example of why it is not perfect. `if (use_mvcc)` is complete - adding `== yes` is redundant. Is there a way to have yes implicitly convert to something truey?

Reply:
No, even with underlying type bool you can't implicitly convert a enum class to a bool, bummer.

Still, for reading code the "explicit bool thing" is very, very useful, imo! Not perfect, but far better than raw bools